File: python/diffrents_types_of_sorting/bubble_sort.py
list=[2,45,42,3,2,4,65,45]
for i in range(len(list)-1):
    for j in range(len(list)-1):
        if list[j]>list[j+1]:
            list[j],list[j+1]=list[j+1],list[j]
                 #OR
            # temp=list[j]
            # list[j]=list[j+1]
            # list[j+1]=temp


print(list)

File: c_lang_program/sort/bubble_sort.c
#include <stdio.h>
int main()
{
    int n,i,j;
    printf("Enter array size ");
    scanf("%d", &n);
    int arr[n];
    printf("Enter values in array \n");

    for (int i = 0; i < n; i++)
    {
        scanf("%d", &arr[i]);
    }


    for(i=n;i>0;i--)
    {
        for(j=0;j<i-1;j++)
        {
            if(arr[j]>arr[j+1])

            {
                int temp=arr[j];
                arr[j]=arr[j+1];
                arr[j+1]=temp;
            }
        }
    }

    printf("sorted result :\n");
    for(i=0;i<n;i++)
    {
        printf("%d ",arr[i]);
    }
    return 0;
}